OPAL/tac/src/main/scala/org/opalj/tac/cg/AndroidCallGraphKey.scala

```diff
@@ -0,0 +1,55 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package cg
+
+import org.opalj.br.analyses.{ProjectInformationKey, SomeProject}
+import org.opalj.br.fpcf.FPCFAnalysisScheduler
+import org.opalj.tac.fpcf.analyses.cg.{CHATypeProvider, RTATypeProvider, TypeProvider, eagerAndroidICCAnalysisScheduler}
+import org.opalj.tac.fpcf.analyses.cg.rta.{ConfiguredNativeMethodsInstantiatedTypesAnalysisScheduler, InstantiatedTypesAnalysisScheduler}
+
+/**
+ * A [[org.opalj.br.analyses.ProjectInformationKey]] to compute a [[CallGraph]] for Android based on the
+ * [[org.opalj.br.analyses.ProjectInformationKey]] Parameter cgk. Currently only supports CHA and RTA.
+ *
+ * @see [[AbstractCallGraphKey]] for further details.
+ *
+ * Does not work for Androidx
+ *
+ * @param cgk Need to specify the key of the call graph algorithm that should be used e.g. CHACallGraphKey
+ * @param manifestPath Path to the projects AndroidManifest.xml
+ *
+ * @author Tom Nikisch
+ */
+class AndroidCallGraphKey(cgk: ProjectInformationKey[CallGraph, Nothing], manifestPath: String) extends CallGraphKey {
+    /**
+     * Lists the call graph specific schedulers that must be run to compute the respective call
+     * graph.
+     *
+     */
+    override protected def callGraphSchedulers(
+        project: SomeProject
+    ): Traversable[FPCFAnalysisScheduler] = {
+        val AndroidICC = eagerAndroidICCAnalysisScheduler
```

Comment:
Use proper camelCase (i.e., start with a small letter) for local variables. Only classes should start with a capital letter.